### PR TITLE
test against 1.9.1

### PR DESCRIPTION
**Pull request by NatalieEO**
Part of https://ucsc-cgl.atlassian.net/browse/SEAB-1614

### CODE REVIEW

**Comment by NatalieEO** [comment]
> You're targeting master... Not sure if the intent is to target develop or the hotfix?
> 
> Also, run-webservice-script.sh doesn't even use this setting; not clear how to resolve that.

Oh I think I misunderstood the second bullet point in the ticket:`update to at least 1.9.1 prod for the master branch tests`
Would that be the hotfix then?

**Comment by coverbeck** [comment]
> Oh I think I misunderstood the second bullet point in the ticket:`update to at least 1.9.1 prod for the master branch tests`
> Would that be the hotfix then?

Hmm, hadn't read that and maybe I misunderstood. But I haven't seen PRs directly against master like this. We already released what's on master and don't run tests against it, so I would think it goes into 1.9.2 hotfix branch, but @denis-yuen should confirm.

**Comment by denis-yuen** [comment]
> Hmm, hadn't read that and maybe I misunderstood. But I haven't seen PRs directly against master like this. We already released what's on master and don't run tests against it, so I would think it goes into 1.9.2 hotfix branch, but @denis-yuen should confirm.

Yeah, I was probably inaccurate. I
I think to be kosher, targeting this at the hot-fix branch makes sense.

**Comment by garyluu** [comment]
Interesting...are the tests not re-triggered when the base is changed?

**Comment by NatalieEO** [comment]
> Interesting...are the tests not re-triggered when the base is changed?
Ah yeah... doesn't look like it. Maybe I should make a new PR with a branch created off of the hotfix?

**Comment by denis-yuen** [comment]
You can also trigger a build by updating the readme.md or something

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "2.3.0",
   "license": "Apache License 2.0",
   "config": {
-    "webservice_version": "1.9.0-beta.0"
+    "webservice_version": "1.9.1"
```
Comment (coverbeck) [inline package.json:6]: Now that this is on 1.9.2 hotfix branch, shouldn't this be 1.9.2? Which isn't out yet? This is a broader question, but how are we suppose to handle this? As soon as we do the 1.9.2 web service release, update this to point to 1.9.2?

Also I had a comment about CircleCI artifacts, which I thought take precedence over this, but Gary approved, so I'm not sure.

Comment (NatalieEO) [reply package.json:6]: I thought the run-webservice-script.sh would use this webservice version as long as it was commented correctly? Unless you meant something else.

And yeah, I don't see a tag for 1.9.2. I guess bare minimum we make sure it was done before release to Dockstore? I don't know where in the release process Gary has those instructions.

Comment (coverbeck) [reply package.json:6]: > I thought the run-webservice-script.sh would use this webservice version as long as it was commented correctly? Unless you meant something else.

You're right; I was looking at a branch that was pointing at CircleCI, but this branch is using the artifactory one.

**Pull request merged**